app/views/roles/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for(:title, "Roles") %>
+
+<% if @roles.empty? %>
+  <div class="none-found">
+    No roles found.
+  </div>
```

Comment:
Nice job covering the zero case. :)

However, in the new scheme, you will always have roles. You may have some roles with no users, which you should not show, but there will always be at least one admin.